cli/bench/http/deno_http_serve_https.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+const addr = Deno.args[0] ?? "127.0.0.1:4500";
+const [hostname, port] = addr.split(":");
+const { serve } = Deno;
+
+const CERT = `-----BEGIN CERTIFICATE-----
+MIIDGjCCAgICCQC0tTR5lER3XDANBgkqhkiG9w0BAQsFADBPMQswCQYDVQQGEwJD
```

Comment:
Where are these from? We have testing keys checked into the repo.

Reply:
Good catch, I see quite a few testing keys I can make use of.